tests/cargo-ui/ws-package-exclude-unknown/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+# Copyright Kani Contributors
```

Comment:
The tests seem to be missing the expected files.

Reply:
Good catch! Added them.